.github/workflows/main.yml

```diff
@@ -48,14 +48,36 @@ jobs:
             task: r-package
             compiler: clang
             r_version: 4.0
+          - os: windows-latest
+            task: r-package
+            compiler: MINGW
+            toolchain: MINGW
+            r_version: 3.6
+          - os: windows-latest
+            task: r-package
+            compiler: MINGW
+            toolchain: MSYS
+            r_version: 4.0
+          # Visual Studio 2017
+          - os: windows-2016
+            task: r-package
+            compiler: MSVC
+            toolchain: MSVC
+            r_version: 3.6
+          # Visual Studio 2019
+          - os: windows-2019
+            task: r-package
+            compiler: MSVC
+            toolchain: MSVC
+            r_version: 4.0
     steps:
       - name: Checkout repository
         uses: actions/checkout@v1
         with:
           fetch-depth: 5
           submodules: true
       - name: Setup and run tests on Linux and macOS
-        if: matrix.os != 'windows-latest'
+        if: matrix.os == 'macOS-latest' || matrix.os == 'ubuntu-latest'
         shell: bash
         run: |
           export TASK="${{ matrix.task }}"
```

Comment:
Please fix trailing `;` in the following lines
https://github.com/microsoft/LightGBM/blob/e83042f20633d7f74dda0d18624721447a610c8b/.github/workflows/main.yml#L65-L69

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3168/commits/6402769a3c297c9825855639029ff68af63c9b3d